test/JDBC/expected/TestDecimal-vu-verify.out

```diff
@@ -265,3 +265,11 @@ select in3 * in4 from testdecimal_vu_prepare_tab2;
 
 ~~ERROR (Message: Arithmetic overflow error for data type numeric.)~~
 
+
+#BABEL-3066
+SELECT CAST(123456.25 AS real) AS value_as_real, CAST(CAST(123456.25 AS real) AS decimal(30,2)) AS value_casted_to_decimal_30_2;
+~~START~~
+real#!#numeric
+123456.25#!#123456.25
```

Comment:
Could you add a test case like : 

select distinct t from a group by CAST(CAST(a.t AS real) AS decimal(30,2))

or something like 

and add a test case like 

insert into a table values ( cast xxx) 

then explain select xxx from table a ( Just wanted to take a look at whether it's index only scan )